src/UI/Implementation/Component/Panel/Factory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Panel;
+
+use ILIAS\UI\Component\Panel as P;
+
+/**
+ * Class Factory
```

Comment:
Just for curiosity: Is this a line PHPStorm always creates? I stumbled upon it very often already, as i consider this non-informative.